client/my-sites/woocommerce/woop/style.scss

```
@@ -20,3 +34,66 @@ body.is-section-woocommerce-installation.theme-default.color-scheme {
 		}
 	}
 }
+
+.woop__landing-page-features-section {
+	@include woocommerce-colors();
+	margin-top: 100px;
+	padding-top: 60px;
+	padding-bottom: 40px;
+	padding-left: 20px;
+    padding-right: 20px;
+
+	// on desktop make the section full width so it takes the whole space
+	@media ( min-width: 660px ) {
+		margin-left: -30px;
+		width: calc( 100% + 20px );
+	}
+
+	background-color: var( --color-woocommerce-onboarding-background );
+	.formatted-header .formatted-header__title {
+
+		@media ( max-width: 480px ) {
+			margin-left: 0;
+			margin-right: 0;
+			width: 100%;
+			max-width: fit-content;
+			text-align: center;
+		}
+
+		text-align: left;
+		margin-left: 32px;
+		font-size: 2rem;
+		max-width: 377px;
+		line-height: 40px;
+		font-family: 'Recoleta';
```

There's a variable for this font:
```suggestion
		@include onboarding-font-recoleta;
```